src/views/Project/index.jsx

```diff
@@ -111,12 +136,17 @@ export default class Project extends Component {
       [];
 
     return (
-      <Fragment>
-        <header>
+      <div className={classes.root}>
+        <AppBar position="absolute" className={classes.header}>
+          <IconButton className={classes.link} component={NavLink} to="/">
+            <ArrowLeftIcon />
+          </IconButton>
           <Typography variant="display2" align="center">
             {project.name}
           </Typography>
-          <Typography variant="subheading" align="center">
+        </AppBar>
+        <div className={classes.container}>
+          <Typography variant="subheading" align="center" color="textSecondary">
```

Comment:
Now that we are using an app bar on top, instead of defining "Bugs & Issues" here, I think we can make it neater by defining a prop a new prop on `DataTable` called `title` and do something like:

```jsx
<Fragment>
  {title && (
    <Toolbar>
      <Typography variant="title">
        {title}
      </Typography>
    </Toolbar>
  )}
  // ...
</Fragment>
```

Then you can do

```jsx
<DataTable
  title="Bugs & Issues"
  ... />
```